HISTORY.md

```diff
@@ -10,6 +10,8 @@
 * Add transaction `get_pinned` and `multi_get` to C API.
 * Add two-phase commit support to C API.
 * Add `rocksdb_transaction_get_writebatch_wi` and `rocksdb_transaction_rebuild_from_writebatch` to C API.
+* Add `rocksdb_options_get_blob_file_starting_level` and `rocksdb_options_set_blob_file_starting_level` to C API.
+* Add `Java_org_rocksdb_Options_blobFileStartingLevel` and `Java_org_rocksdb_Options_setBlobFileStartingLevel` to Java API.
```

Comment:
We would want to put the user-facing method names here (like `blobFileStartingLevel` / `setBlobFileStartingLevel`)

Reply:
done